core/dvector.h

```diff
@@ -466,6 +467,16 @@ class PoolVector {
 		return OK;
 	}
 
+	String join(String delimiter) {
```

Comment:
I think this should be stubbed here, and implemented in the C++ code, for faster recompilation (since it needs not to be inline)